apps/percona/chaos/storage_data_replica_failure/chaosutil.j2

```diff
@@ -0,0 +1,5 @@
+{% if sc is defined and sc == 'openebs-standard' %}
```

Comment:
name of this file .. can be openebs volume replica failure. -- storage data replica .. might mean percona master/replica.

Reply:
Done, @kmova